packages/web-components/fast-components/src/anchored-region/resize-observer.ts

```diff
@@ -1,17 +1,17 @@
-import { ResizeObserverEntry } from "./resize-observer-entry";
+// import { ResizeObserverEntry } from "./resize-observer-entry";
 
-export type ConstructibleResizeObserver = new (
-    callback: ResizeObserverCallback
-) => ResizeObserverClassDefinition;
+// export type ConstructibleResizeObserver = new (
```

Comment:
Can this file be removed?

Reply:
yes, there was a redundant copy of those, gone.